services/consensuscontext/helpers.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 the orbs-network-go authors
+// This file is part of the orbs-network-go library in the Orbs project.
+//
+// This source code is licensed under the MIT license found in the LICENSE file in the root directory of this source tree.
+// The above notice should be included in all copies or substantial portions of the software.
+
+package consensuscontext
+
+import (
+	"context"
+	"github.com/orbs-network/orbs-network-go/crypto/digest"
+	"github.com/orbs-network/orbs-network-go/instrumentation/logfields"
+	"github.com/orbs-network/orbs-spec/types/go/primitives"
+	"github.com/orbs-network/orbs-spec/types/go/protocol"
+	"github.com/orbs-network/orbs-spec/types/go/services"
+	"github.com/orbs-network/scribe/log"
+	"github.com/pkg/errors"
+)
+
+func (s *service) fixPrevReferenceTimeIfGenesis(ctx context.Context, blockHeight primitives.BlockHeight, prevBlockReferenceTime primitives.TimestampSeconds) (primitives.TimestampSeconds, error) {
+	if blockHeight == 1 { // genesis block
```

Comment:
idiomatically reverse the condition?

Reply:
if i reverse then i need to add another return ... and its not an error , its special case. i prefer it this way.